### ISSUE TITLE
DOCUMENTATION: Could we add translate author in blog attribution

### ISSUE THREAD

**Issue by wilsonwu** [istio/istio.io#13454]
Such as:

```
title: title
subtitle: subtitle
description: description
publishdate: 2023-06-25
attribution: First Last (Company), Translator: First Last (Company)
keywords: [keyword]
```

Could we add translator name and company to the attribution, and for not en version of blogs, Translator: First Last (Company) will be [zh] 译者：姓名（公司）, etc.

**Comment by wilsonwu** [comment]
Do we have any suggestion for this improvement?

**Comment by ericvn** [comment]
I am fine with this, but I think this question might be better for the @istio/steering-committee to answer since I think they have more control of the blogs.

**Comment by craigbox** [comment]
I support the concept: would it make more sense to have a different piece of front matter for the translator for a particular version, and have it used in the template if it is set?

<img width="809" alt="Screenshot 2023-07-03 at 9 40 46 PM" src="https://github.com/istio/istio.io/assets/132510/72fe50d9-a4ca-4500-8260-7510508ae7df">

**Comment by hzxuzhonghu** [comment]
I am concerned about this because sometimes we may frequently update the blog.  There is already git history, from that users can find out the author/translator easily.

**Comment by wilsonwu** [comment]
> I am concerned about this because sometimes we may frequently update the blog. There is already git history, from that users can find out the author/translator easily.

Yes, this is true for community users, but for normal users, if the translator's information can be displayed in the blog, it will give more encouragement to the translator.

**Comment by wilsonwu** [comment]
> Translator

If just add translated by in the attribution field, is that OK? like:
```
attribution: First Last (Company), Translated by: First Last (Company)
```

Do you think so? @craigbox

**Comment by craigbox** [comment]
I would rather see it done the right way in the long run, but I wouldn't be upset by it being there.  Please use a semicolon (`;`) between the two.

(To be honest I wouldn't see it, I'm not a /zh/ approver 😁)

**Comment by wilsonwu** [comment]
There is a pilot: https://github.com/istio/istio.io/pull/13529, if there is no more suggestion, after the pilot be merged, I will close this issue, for more questions let's start new discussion.

**Comment by windsonsea** [comment]
Seems this is a good choice as merged in #13529 right now:

```
attribution: First Last (Company); Translated by: First Last (Company)
```
